plasmapy/formulary/magnetostatics.py

```diff
@@ -458,28 +492,92 @@ def magnetic_field(self, p) -> u.T:
 
         """
 
-        x, w = self.roots_legendre
-        t = x * np.pi
-        pt = self.curve(t)
-        dl = self.radius * (
-            -np.matmul(np.expand_dims(self.axis_x, 1), np.expand_dims(np.sin(t), 0))
-            + np.matmul(np.expand_dims(self.axis_y, 1), np.expand_dims(np.cos(t), 0))
-        )  # (3, n)
+        field = np.zeros((p.shape[0], pt.shape[1], p.shape[1]))
+        for i in numba.prange(pt.shape[1]):  # 300
```

Comment:
I'm not sure what the `# 300` refers to.

Reply:
The Spartans, obviously! Nah, it's spilled debug juice from when I was trying to rewrite it in Numba. I'll remove it - thanks for pointing it out :)